getting-started/kubernetes/hardway/install-cni-plugin.md

```diff
@@ -178,6 +178,19 @@ cat > /etc/cni/net.d/10-calico.conflist <<EOF
 EOF
 ```
 
+Step back to normal user.
+
+```bash
+exit
+```
+
+At this point Kubernetes nodes must became to `Ready` state, because Kubernetes has all the network configuration.
+
+Write the CNI configuration
```

Comment:
Hmm I think this comment is out of place. I'd just remove this since we already mention it earlier.